aws/README.md

```diff
@@ -134,6 +134,22 @@ Deploying to `prd` requires a [manual action](https://docs.gitlab.com/ce/ci/yaml
 
 See [CI-CD.md](../CI-CD.md)
 
+## Production is broken, there's been a security breach, or there is some other operational emergency
+
+If you need to contact a human on the Ops team urgently, here are steps you can try. They escalate in urgency and disruptiveness, so favor earlier steps and patience, but use your best judgement. If production is down right before a big demo or an attack is happening now, it's more important to get an engineer's attention than it is to avoid sending notifications to a few extra people.
+
```

Comment:
What happened to "don't panic!" point? :)

Reply:
If this is a serious suggestion, can you propose some encouraging text?

Reply:
I am pretty bad at encouraging, but I am serious – from my experience, in case of real production failure this helps.

Reply:
Ok. I'll take a crack at this over in https://github.com/gpii-ops/gpii-infra/pull/75, which happens to contain some other doc fixes I made today.